### PR TITLE
WindowOcclusion shader and test scene

### PR DESCRIPTION
**Pull request by EricRehmeyer**
Added a WindowOcclusion shader for the already-published case study, plus a simple scene to demonstrate it working.

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Shaders/WindowOcclusion.shader [hunk 1]
```diff
@@ -0,0 +1,51 @@
+///
+/// Simple occlusion shader that can be used to hide other objects.
+/// This prevents other objects from being rendered by drawing invisible 'opaque' pixels to the depth buffer.
+///
+Shader "HoloToolkit/WindowOcclusion"
+{
+    Properties
```
Comment (NeerajW) [inline Assets/HoloToolkit/Utilities/Shaders/WindowOcclusion.shader]: Delete empty Properties?

File: README.md [hunk 2]
```diff
@@ -579,6 +579,10 @@ This scene shows how to use ManualCameraControl.cs.  The script is on the main c
 
 This scene demonstrates how to use TextToSpeechManager.cs.  The script is placed on 3 cubes in the scene. Whenever a cube is activated with an air tap, a text to speech voice will emanate from the cube. The user can also ask "What time is it?" to hear the current time from a voice that stays with the user as they move.
 
+#### WindowOcclusion.unity
```
Comment (NeerajW) [inline README.md]: Please update the description for the Shader as well. Specially, the difference between Occlusion and WindowOcclusion.

File: Assets/HoloToolkit/Utilities/Shaders/WindowOcclusion.shader (hunk 1 above)
Comment (EricRehmeyer) [inline Assets/HoloToolkit/Utilities/Shaders/WindowOcclusion.shader]: Not sure if that'd work, but it's what Occlusion.shader does, which is already in HoloToolkit-Unity.

Comment (NeerajW) [inline Assets/HoloToolkit/Utilities/Shaders/WindowOcclusion.shader]: Agreed, thanks. I recalled and then saw your comment.

**Pull request merged**